### DISCUSSION TITLE
How to use download_metadata via API_KEY?

### DISCUSSION THREAD

**Discussion by houmie** [fastlane/fastlane#22171] [Q&A]
Previously I was able to download the existing meta data from AppStore like this:

```
bundle exec fastlane deliver download_metadata --skip_screenshots
```

But this no longer works, as it fails with the error:
```
[!] The request could not be completed because:
	Access forbidden
```

I believe I need to use my API key like I have set it up in Fastfile:

```
default_platform(:ios)

platform :ios do
  desc "Push a new release build to the App Store"
  lane :release do
    api_key = app_store_connect_api_key(
        key_id: "xxx",
        issuer_id: "xxxx",
        key_filepath: "/Users/houmie/Documents/.AuthKey_xxx.p8",
        duration: 1200,
    )
    build_app(scheme: "iOS-Release")
    upload_to_app_store
  end
end
```

Any idea how I could achieve this, please?

**Comment by sbeitzel** [comment]
So, this is a little difficult to find, but there are clues in the documentation. The short answer is:

1. Create a JSON file with your API key in it
2. invoke deliver on the command line with a parameter specifying your JSON key file

`fastlane deliver download_metadata --skip_screenshots --api_key_path /Users/houmie/Documents/.AuthKey_xxx.json`

The hint is at the very bottom of the [App Store Connect API](https://docs.fastlane.tools/app-store-connect-api/) page.
